### ISSUE TITLE
enable and disable editing

### ISSUE THREAD

**Issue by omidaminiva** [AvinashSKaranth/android-summernote#4]
is it possible to add a method or flag to enable and disable editing?

**Comment by AvinashSKaranth** [comment]
Update your Fragment/Activity to show in WebView instead of summernote when disabled editing.

**Comment by omidaminiva** [comment]
Yes I already have a webview to do that, but I do not want the user see loading again and again. I want to create a seamless experience for user.
